src/server/services/wms/qgswms.cpp

```diff
@@ -81,6 +81,14 @@ namespace QgsWms
           version = mVersion;
         }
 
+        // WMS 1.3.0 specification: If a version lower than any of those known
+        // to the server is requested, then the server shall send the lowest
+        // version it supports.
+        if ( QgsProjectVersion( 1, 1, 1 ) > parameters.versionAsNumber() )
+        {
+          version = "1.1.1";
```

Comment:
```suggestion
          version = QStringLiteral( "1.1.1" );
```

Just because we think that it looks pro if things are more complicated than they could potentially be :stuck_out_tongue_closed_eyes:

Reply:
It was just a quick and dirty PR to see if OGC tests and unit tests were still green with this modification, but thanks :D